src/library/VTKExtensions/Core/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/vtkF3DConfigure.h.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/vtkF3DConfigure.h"
+  @ONLY)
+
+set(classes
+  F3DLog
+  F3DOffscreenRender
```

Comment:
Non VTK classes in VTK module looks weird too me.

Reply:
three possibilities:

- Keep as is (but maybe better naming?)
- Make F3D classes vtkF3D classes but not inheriting from vtkObject. This is fair and already exists in VTK like that.
- Make these classes inherit vtkObject, useless
- Put these classes into their own non-vtk-module-but-vtk-dependant small library. Way too much work for no benefit

1 and 2 are fine by me. wdyt ?

Reply:
1 is fine for now, let's see later if we can improve it

Reply:
Any suggestions for names ?

Current situation:
f3d_* : public API
vtkF3D : vtk modules
F3D : internal stuff + vtk modules stuff


Suggestion:
1.

f3d_* : public API
f3d_itnl_* : internal stuff
vtkF3D : vtk modules
F3D : vtk modules stuff non vtk sutff

2.
f3d_* : public API
F3D* : internal stuff
vtkF3D : vtk modules
vF3D : vtk modules stuff non vtk sutff

Something else ?

Reply:
Closing this discussion as we decided offline